src/reducers/cartReducer.js

```diff
@@ -1,12 +1,81 @@
+import { SET_QUANTITY, REMOVE_ITEM, INCREASE_QUANTITY, DECREASE_QUANTITY } from '../constants/actionTypes';
+
 const initialState = {
-  items: []
+  items: [
+    {
+      id: 'EOK3293KSD',
+      name: "Men's Chaps Performance Series Classic-Fit 4-Way Strech Sit Jacket",
+      imgUrl: 'https://images-na.ssl-images-amazon.com/images/I/71ZgPluVoTL._UX385_.jpg',
+      variants: {
+        size: '42 SHORT',
+        color: 'Charcoal'
+      },
+      price: {
+        regular: 240,
+        discount: 125
+      },
+      quantity: 1
+    },
+    {
+      id: 'EOK32943983',
+      name: "WoMen's Chaps Performance Series Classic-Fit 4-Way Strech Sit Jacket",
+      imgUrl: 'https://n.nordstrommedia.com/ImageGallery/store/product/Zoom/6/_103433446.jpg',
+      variants: {
+        size: '42 SHORT',
+        color: 'Charcoal'
+      },
+      price: {
+        regular: 440,
+        discount: 140
+      },
+      quantity: 1
+    }
+  ]
+};
+
+const getItemQuantity = function(array, itemId) {
+  const item = array.find(item => item.id === itemId);
+  return item ? item.quantity : null;
+};
+
+const updateItemQuantity = function(array, itemId, quantity) {
+  return array.map(item => {
```

Comment:
If you want to change only one instance of Item, use `findIndex` instead of `map`.

Reply:
But I also do not want to immute the state. I find this way in Redux docs: https://redux.js.org/recipes/structuring-reducers/immutable-update-patterns#updating-an-item-in-an-array